File: backend/models/location.go
package models

type Location struct {
	ID         uint   `gorm:"primary_key"`
	Street     string `json:"street"`
	Number     string `json:"number"`
	City       string `json:"city"`
	Country    string `json:"country"`
	PostalCode string `json:"postal_code"`
	UserID     uint   `json:"user_id"`
}

File: frontend/src/app/models/location.ts
export interface Location {
    ID: number;
    street: string;
    number: string;
    city: string;
    country: string;
    postal_code: string;
    user_id: number;
};

export interface CreateLocation {
    street: string;
    number: string;
    city: string;
    country: string;
    postal_code: string;
};